Gemfile

```diff
@@ -13,14 +12,16 @@ end
 # used by dummy application
 group :development, :test do
   # Upload coverage reports to coveralls.io
-  gem 'coveralls', require: false  
+  gem 'coveralls', require: false
   # supplies factories for producing model instance for specs
   # Version 4.1.0 or newer is needed to support generate calls without the 'FactoryBot.' in factory definitions syntax.
   gem 'factory_bot'
   # auto-load factories from spec/factories
   gem 'factory_bot_rails'
 
   gem 'rails', '~> 6.0'
+  gem 'net-smtp', require: false
```

Comment:
Required otherwise the tests fails with a rails bump

From CI:

```
rake, version 13.0.6
You don't have net-smtp installed in your application. Please add it to your Gemfile and run bundle install
Created database 'metasploit_data_models_test'
```

Test failures:

```
  431) Mdm::Host validations arch with known architecture 'mipsbe' is expected to be valid
       Failure/Error: required = zeitwerk_original_require(path)

       LoadError:
         cannot load such file -- net/smtp
       # /home/runner/work/metasploit_data_models/metasploit_data_models/vendor/bundle/ruby/3.1.0/gems/zeitwerk-2.6.0/lib/zeitwerk/kernel.rb:35:in `require'
       # /home/runner/work/metasploit_data_models/metasploit_data_models/vendor/bundle/ruby/3.1.0/gems/zeitwerk-2.6.0/lib/zeitwerk/kernel.rb:35:in `require'
       # /home/runner/work/metasploit_data_models/metasploit_data_models/vendor/bundle/ruby/3.1.0/gems/mail-2.7.1/lib/mail.rb:9:in `<module:Mail>'
       # /home/runner/work/metasploit_data_models/metasploit_data_models/vendor/bundle/ruby/3.1.0/gems/mail-2.7.1/lib/mail.rb:3:in `<top (required)>'
       # /home/runner/work/metasploit_data_models/metasploit_data_models/vendor/bundle/ruby/3.1.0/gems/zeitwerk-2.6.0/lib/zeitwerk/kernel.rb:35:in `require'
       # /home/runner/work/metasploit_data_models/metasploit_data_models/vendor/bundle/ruby/3.1.0/gems/zeitwerk-2.6.0/lib/zeitwerk/kernel.rb:35:in `require'
       # /home/runner/work/metasploit_data_models/metasploit_data_models/vendor/bundle/ruby/3.1.0/gems/actionmailer-6.1.7/lib/action_mailer/mail_with_error_handling.rb:4:in `<top (required)>'
       # /home/runner/work/metasploit_data_models/metasploit_data_models/vendor/bundle/ruby/3.1.0/gems/zeitwerk-2.6.0/lib/zeitwerk/kernel.rb:35:in `require'
       # /home/runner/work/metasploit_data_models/metasploit_data_models/vendor/bundle/ruby/3.1.0/gems/zeitwerk-2.6.0/lib/zeitwerk/kernel.rb:35:in `require'
       # /home/runner/work/metasploit_data_models/metasploit_data_models/vendor/bundle/ruby/3.1.0/gems/actionmailer-6.1.7/lib/action_mailer/base.rb:3:in `<top (required)>'
       # /home/runner/work/metasploit_data_models/metasploit_data_models/vendor/bundle/ruby/3.1.0/gems/zeitwerk-2.6.0/lib/zeitwerk/kernel.rb:35:in `require'
       # /home/runner/work/metasploit_data_models/metasploit_data_models/vendor/bundle/ruby/3.1.0/gems/zeitwerk-2.6.0/lib/zeitwerk/kernel.rb:35:in `require'
       # /home/runner/work/metasploit_data_models/metasploit_data_models/vendor/bundle/ruby/3.1.0/gems/rspec-rails-5.1.2/lib/rspec/rails/configuration.rb:174:in `block in initialize_configuration'
       # /home/runner/work/metasploit_data_models/metasploit_data_models/vendor/bundle/ruby/3.1.0/gems/rspec-core-3.11.0/lib/rspec/core/example.rb:457:in `instance_exec'
       # /home/runner/work/metasploit_data_models/metasploit_data_models/vendor/bundle/ruby/3.1.0/gems/rspec-core-3.11.0/lib/rspec/core/example.rb:457:in `instance_exec'
       # /home/runner/work/metasploit_data_models/metasploit_data_models/vendor/bundle/ruby/3.1.0/gems/rspec-core-3.11.0/lib/rspec/core/hooks.rb:372:in `run'
       # /home/runner/work/metasploit_data_models/metasploit_data_models/vendor/bundle/ruby/3.1.0/gems/rspec-core-3.11.0/lib/rspec/core/hooks.rb:529:in `block in run_owned_hooks_for'
       # /home/runner/work/metasploit_data_models/metasploit_data_models/vendor/bundle/ruby/3.1.0/gems/rspec-core-3.11.0/lib/rspec/core/hooks.rb:528:in `each'
       # /home/runner/work/metasploit_data_models/metasploit_data_models/vendor/bundle/ruby/3.1.0/gems/rspec-core-3.11.0/lib/rspec/core/hooks.rb:528:in `run_owned_hooks_for'
       # /home/runner/work/metasploit_data_models/metasploit_data_models/vendor/bundle/ruby/3.1.0/gems/rspec-core-3.11.0/lib/rspec/core/hooks.rb:615:in `block in run_example_hooks_for'
       # /home/runner/work/metasploit_data_models/metasploit_data_models/vendor/bundle/ruby/3.1.0/gems/rspec-core-3.11.0/lib/rspec/core/hooks.rb:614:in `each'
       # /home/runner/work/metasploit_data_models/metasploit_data_models/vendor/bundle/ruby/3.1.0/gems/rspec-core-3.11.0/lib/rspec/core/hooks.rb:614:in `run_example_hooks_for'
       # /home/runner/work/metasploit_data_models/metasploit_data_models/vendor/bundle/ruby/3.1.0/gems/rspec-core-3.11.0/lib/rspec/core/hooks.rb:485:in `run'
       # /home/runner/work/metasploit_data_models/metasploit_data_models/vendor/bundle/ruby/3.1.0/gems/rspec-core-3.11.0/lib/rspec/core/example.rb:518:in `run_after_example'
       # /home/runner/work/metasploit_data_models/metasploit_data_models/vendor/bundle/ruby/3.1.0/gems/rspec-core-3.11.0/lib/rspec/core/example.rb:283:in `block in run'
       # /home/runner/work/metasploit_data_models/metasploit_data_models/vendor/bundle/ruby/3.1.0/gems/rspec-core-3.11.0/lib/rspec/core/example.rb:511:in `block in with_around_and_singleton_context_hooks'
       # /home/runner/work/metasploit_data_models/metasploit_data_models/vendor/bundle/ruby/3.1.0/gems/rspec-core-3.11.0/lib/rspec/core/example.rb:468:in `block in with_around_example_hooks'
       # /home/runner/work/metasploit_data_models/metasploit_data_models/vendor/bundle/ruby/3.1.0/gems/rspec-core-3.11.0/lib/rspec/core/hooks.rb:486:in `block in run'
       # /home/runner/work/metasploit_data_models/metasploit_data_models/vendor/bundle/ruby/3.1.0/gems/rspec-core-3.11.0/lib/rspec/core/hooks.rb:626:in `block in run_around_example_hooks_for'
       # /home/runner/work/metasploit_data_models/metasploit_data_models/vendor/bundle/ruby/3.1.0/gems/rspec-core-3.11.0/lib/rspec/core/example.rb:352:in `call'
       # /home/runner/work/metasploit_data_models/metasploit_data_models/vendor/bundle/ruby/3.1.0/gems/rspec-rails-5.1.2/lib/rspec/rails/adapters.rb:75:in `block (2 levels) in <module:MinitestLifecycleAdapter>'
       # /home/runner/work/metasploit_data_models/metasploit_data_models/vendor/bundle/ruby/3.1.0/gems/rspec-core-3.11.0/lib/rspec/core/example.rb:457:in `instance_exec'
       # /home/runner/work/metasploit_data_models/metasploit_data_models/vendor/bundle/ruby/3.1.0/gems/rspec-core-3.11.0/lib/rspec/core/example.rb:457:in `instance_exec'
       # /home/runner/work/metasploit_data_models/metasploit_data_models/vendor/bundle/ruby/3.1.0/gems/rspec-core-3.11.0/lib/rspec/core/hooks.rb:390:in `execute_with'
       # /home/runner/work/metasploit_data_models/metasploit_data_models/vendor/bundle/ruby/3.1.0/gems/rspec-core-3.11.0/lib/rspec/core/hooks.rb:628:in `block (2 levels) in run_around_example_hooks_for'
       # /home/runner/work/metasploit_data_models/metasploit_data_models/vendor/bundle/ruby/3.1.0/gems/rspec-core-3.11.0/lib/rspec/core/example.rb:352:in `call'
       # /home/runner/work/metasploit_data_models/metasploit_data_models/vendor/bundle/ruby/3.1.0/gems/rspec-core-3.11.0/lib/rspec/core/hooks.rb:629:in `run_around_example_hooks_for'
       # /home/runner/work/metasploit_data_models/metasploit_data_models/vendor/bundle/ruby/3.1.0/gems/rspec-core-3.11.0/lib/rspec/core/hooks.rb:486:in `run'
       # /home/runner/work/metasploit_data_models/metasploit_data_models/vendor/bundle/ruby/3.1.0/gems/rspec-core-3.11.0/lib/rspec/core/example.rb:468:in `with_around_example_hooks'
       # /home/runner/work/metasploit_data_models/metasploit_data_models/vendor/bundle/ruby/3.1.0/gems/rspec-core-3.11.0/lib/rspec/core/example.rb:511:in `with_around_and_singleton_context_hooks'
       # /home/runner/work/metasploit_data_models/metasploit_data_models/vendor/bundle/ruby/3.1.0/gems/rspec-core-3.11.0/lib/rspec/core/example.rb:259:in `run'
       # /home/runner/work/metasploit_data_models/metasploit_data_models/vendor/bundle/ruby/3.1.0/gems/rspec-core-3.11.0/lib/rspec/core/example_group.rb:646:in `block in run_examples'
       # /home/runner/work/metasploit_data_models/metasploit_data_models/vendor/bundle/ruby/3.1.0/gems/rspec-core-3.11.0/lib/rspec/core/example_group.rb:642:in `map'
       # /home/runner/work/metasploit_data_models/metasploit_data_models/vendor/bundle/ruby/3.1.0/gems/rspec-core-3.11.0/lib/rspec/core/example_group.rb:642:in `run_examples'
       # /home/runner/work/metasploit_data_models/metasploit_data_models/vendor/bundle/ruby/3.1.0/gems/rspec-core-3.11.0/lib/rspec/core/example_group.rb:607:in `run'
       # /home/runner/work/metasploit_data_models/metasploit_data_models/vendor/bundle/ruby/3.1.0/gems/rspec-core-3.11.0/lib/rspec/core/example_group.rb:608:in `block in run'
       # /home/runner/work/metasploit_data_models/metasploit_data_models/vendor/bundle/ruby/3.1.0/gems/rspec-core-3.11.0/lib/rspec/core/example_group.rb:608:in `map'
       # /home/runner/work/metasploit_data_models/metasploit_data_models/vendor/bundle/ruby/3.1.0/gems/rspec-core-3.11.0/lib/rspec/core/example_group.rb:608:in `run'
       # /home/runner/work/metasploit_data_models/metasploit_data_models/vendor/bundle/ruby/3.1.0/gems/rspec-core-3.11.0/lib/rspec/core/example_group.rb:608:in `block in run'
       # /home/runner/work/metasploit_data_models/metasploit_data_models/vendor/bundle/ruby/3.1.0/gems/rspec-core-3.11.0/lib/rspec/core/example_group.rb:608:in `map'
       # /home/runner/work/metasploit_data_models/metasploit_data_models/vendor/bundle/ruby/3.1.0/gems/rspec-core-3.11.0/lib/rspec/core/example_group.rb:608:in `run'
       # /home/runner/work/metasploit_data_models/metasploit_data_models/vendor/bundle/ruby/3.1.0/gems/rspec-core-3.11.0/lib/rspec/core/example_group.rb:608:in `block in run'
       # /home/runner/work/metasploit_data_models/metasploit_data_models/vendor/bundle/ruby/3.1.0/gems/rspec-core-3.11.0/lib/rspec/core/example_group.rb:608:in `map'
       # /home/runner/work/metasploit_data_models/metasploit_data_models/vendor/bundle/ruby/3.1.0/gems/rspec-core-3.11.0/lib/rspec/core/example_group.rb:608:in `run'
       # /home/runner/work/metasploit_data_models/metasploit_data_models/vendor/bundle/ruby/3.1.0/gems/rspec-core-3.11.0/lib/rspec/core/runner.rb:121:in `block (3 levels) in run_specs'
       # /home/runner/work/metasploit_data_models/metasploit_data_models/vendor/bundle/ruby/3.1.0/gems/rspec-core-3.11.0/lib/rspec/core/runner.rb:121:in `map'
       # /home/runner/work/metasploit_data_models/metasploit_data_models/vendor/bundle/ruby/3.1.0/gems/rspec-core-3.11.0/lib/rspec/core/runner.rb:121:in `block (2 levels) in run_specs'
       # /home/runner/work/metasploit_data_models/metasploit_data_models/vendor/bundle/ruby/3.1.0/gems/rspec-core-3.11.0/lib/rspec/core/configuration.rb:2068:in `with_suite_hooks'
       # /home/runner/work/metasploit_data_models/metasploit_data_models/vendor/bundle/ruby/3.1.0/gems/rspec-core-3.11.0/lib/rspec/core/runner.rb:116:in `block in run_specs'
       # /home/runner/work/metasploit_data_models/metasploit_data_models/vendor/bundle/ruby/3.1.0/gems/rspec-core-3.11.0/lib/rspec/core/reporter.rb:74:in `report'
       # /home/runner/work/metasploit_data_models/metasploit_data_models/vendor/bundle/ruby/3.1.0/gems/rspec-core-3.11.0/lib/rspec/core/runner.rb:115:in `run_specs'
       # /home/runner/work/metasploit_data_models/metasploit_data_models/vendor/bundle/ruby/3.1.0/gems/rspec-core-3.11.0/lib/rspec/core/runner.rb:89:in `run'
       # /home/runner/work/metasploit_data_models/metasploit_data_models/vendor/bundle/ruby/3.1.0/gems/rspec-core-3.11.0/lib/rspec/core/runner.rb:71:in `run'
       # /home/runner/work/metasploit_data_models/metasploit_data_models/vendor/bundle/ruby/3.1.0/gems/rspec-core-3.11.0/lib/rspec/core/runner.rb:45:in `invoke'
       # /home/runner/work/metasploit_data_models/metasploit_data_models/vendor/bundle/ruby/3.1.0/gems/rspec-core-3.11.0/exe/rspec:4:in `<main>'
       # 
       #   Showing full backtrace because every line was filtered out.
       #   See docs for RSpec::Configuration#backtrace_exclusion_patterns and
       #   RSpec::Configuration#backtrace_inclusion_patterns for more information.
```

Reply:
Solution from https://stackoverflow.com/questions/70500220/rails-7-ruby-3-1-loaderror-cannot-load-such-file-net-smtp